schema.sql

```diff
@@ -7,5 +7,6 @@ CREATE TABLE animals (
     escape_attempts INT,
     neutered BOOLEAN,
     weight_kg DECIMAL,
+    species varchar(100),
```

Comment:
It will be great to `alter` the table instead of adding species directly to your already created table. This is because its safe and won't cause any problems to your already established table. To do this you should do something like 
` ALTER TABLE animals ADD species varchar(255)`